packages/contracts/deploy/60_dao-factory.ts

```diff
@@ -0,0 +1,37 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {getContractAddress} from './helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts, ethers} = hre;
+  const {deploy} = deployments;
+
+  const {deployer} = await getNamedAccounts();
+
+  const managingDAOAddress = await getContractAddress('DAO', hre);
```

Comment:
How do we explicitly know that we will be getting the address of the managing DAO? Shouldn't this be a well-known contract deployed once?